geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/Transformers.scala

```diff
@@ -334,6 +334,7 @@ class DateFunctionFactory extends TransformerFunctionFactory {
   val basicDateTimeNoMillis       = StandardDateParser("basicDateTimeNoMillis", ISODateTimeFormat.basicDateTimeNoMillis().withZoneUTC())
   val dateHourMinuteSecondMillis  = StandardDateParser("dateHourMinuteSecondMillis", ISODateTimeFormat.dateHourMinuteSecondMillis().withZoneUTC())
   val millisToDate                = TransformerFn("millisToDate") { args => new Date(args(0).asInstanceOf[Long]) }
+  val secsToDate                  = TransformerFn("secsToDate") { args => new Date(args(0).asInstanceOf[Int].toLong * 1000L) }
```

Comment:
@nelias-ccri also please update the list of functions in the README. Make sure you pull the latest master and merge it into this branch - things have just been merged.